docs/ref/forms/widgets.txt

```diff
@@ -997,3 +997,11 @@ Composite widgets
                     empty_label=("Choose Year", "Choose Month", "Choose Day"),
                 ),
             )
+
+``ColorInput``
+~~~~~~~~~~~~~~
+
+.. class:: ColorInput
+    * ``input_type``: ``'color'``
+    * ``template_name``:``'django/forms/widgets/color.html'``
+    * Renders as: ``<input type='color' ...>``
```

Comment:
This will need the `.. versionadded:: 5.2` note and align the formating to other widget definitions
```suggestion

.. versionadded:: 5.2

.. class:: ColorInput

    * ``input_type``: ``'color'``
    * ``template_name``:``'django/forms/widgets/color.html'``
    * Renders as: ``<input type='color' ...>``
```